spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/SpannerTemplate.java

```diff
@@ -343,12 +332,10 @@ private void logReadOptions(SpannerReadOptions options, StringBuilder logs) {
 		}
 	}
 
-	private StringBuilder logColumns(String tableName, KeySet keys, Iterable<String> columns) {
-		StringBuilder logSb = new StringBuilder("Executing read on table "
-				+ tableName
-				+ " with keys: "
-				+ keys
-				+ " and columns: ");
+	private StringBuilder logColumns(String tableName, KeySet keys,
+			Iterable<String> columns) {
+		StringBuilder logSb = new StringBuilder("Executing read on table " + tableName
```

Comment:
`logSb` should only be constructed if the `DEBUG` log level is on.

Reply:
the intention is that this `private` function is only to be used when DEBUG is on. Indeed, the only usage of it is inside such an if-statement.

Reply:
Thanks, I missed that context. :)